tasks/push_image.yml

```diff
@@ -0,0 +1,22 @@
+---
+
+- name: "Get ECR Token"
+  shell: "aws ecr get-authorization-token --region {{ wimpy_aws_region }} --output text --query \"authorizationData[].authorizationToken\""
+  when:
+    - docker_build.changed
+    - wimpy_aws_account_id is defined
+  register: wimpy_aws_ecr_password
+
+- name: "Log into private registry"
+  docker_login:
+    registry: "{{ wimpy_docker_registry | default('https://index.docker.io/v1/') }}"
+    email: "{{ wimpy_docker_registry_email | default(none) }}"
+    username: "{{ wimpy_docker_registry_username | default('AWS') }}"
+    password: "{{ wimpy_docker_registry_password | default((wimpy_aws_ecr_password.stdout | b64decode).split(':')[1] if wimpy_aws_ecr_password.changed else '') }}"
```

Comment:
I plan to remove the whole README and create it from scratch, that's why I'm ignoring the README in all my PR's